services/solr/finish

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+/opt/solr/bin/solr stop -p ${SOLR_PORT:-8983}
```

Comment:
Will be interesting to see if this will also stop the embedded zoo keeper service when run in solrcloud mode.  I believe it will since I think the embedded zookeeper service is truly embedded (ie, not a separate process) from the solr server.

Reply:
look forward to your testing results! Thank you @apotek